Automattic/wp-calypso
# Checkout: add ID Wallet (OVO) payment method

This adds Indonesian Wallet (OVO) support to the new Checkout flow. The payment method is currently disabled for both Checkouts, while improvements are made to the refund flow.

<img width="494" alt="Screen Shot 2020-08-28 at 10 42 57 AM" src="https://user-images.githubusercontent.com/942359/91580673-6cfb0380-e91b-11ea-8600-19def2654090.png">

**To Test:**

- `id_wallet` is already available to a12s, but will need to be force if testing a non-a11n account
- set your currency to IDR (Indonesia)
- visit Checkout
- verify that OVO is shown as a payment method
- verify that name, NIK, and phone inputs (with Indonesia pre-selected) are shown
- verify that all 3 fields are required
- submit a payment with any name, 16-digit NIK, and 10-digit phone number
- verify that you're redirected to dLocal's sandbox payment gateway

Fixes https://github.com/Automattic/wp-calypso/issues/44832
